models/h2o_dai_regression_auto_insurance/README.md

```diff
@@ -0,0 +1,104 @@
+# H2O Driverless AI (DAI) Scoring Pipeline Example
+
+**Dataset Used** - This example uses the Auto Insurance example originally sourced from from
+[Emcien](https://www.sixtusdakurah.com/resources/The_Application_of_Regularization_in_Modelling_Insurance_Claims.pdf).
+
+
+The prepared dataset file used in this example is available in the
+[notebooks/datasets folder](../../notebooks/datasets/auto_insurance_claims_dataset.csv)
+
+
+# Train and Download the Scoring Pipeline
+
+1. Clone this repository.
+
+2. Upload the dataset in this repository from [../../notebooks/datasets/auto_insurance_claims_dataset.csv](../../notebooks/datasets/auto_insurance_claims_dataset.csv) to
+H2O's Driverless AI and use it to auto-learn an ML model.
+
+3. Select 'Click for Actions' and then 'Predict'.
+
+4. Set the target column to 'outcome' and launch the experiment.
+
+5. After completion of the Driverless AI Experiment, download the scoring pipeline that you want to scan with Certifai.
+
+  Download the [Mojo Scoring Pipeline](https://s3.amazonaws.com/artifacts.h2o.ai/releases/ai/h2o/dai/rel-1.8.5-64/docs/userguide/scoring-mojo-scoring-pipeline.html#mojo-scoring-pipeline-files).
+
+# Setup the Certifai Scanning Environment
+
+1. Follow the [Toolkit Setup](https://cognitivescale.github.io/cortex-certifai/docs/about) in the Certifai documentation to install the Certifai toolkit into a conda
+environment (e.g. `certifai`).
+
+# Scan H2O Mojo Scoring Pipeline
+## Setup the Prediction Service
+
+1. In another terminal, setup a different conda environment (e.g. `daimodel`) to run the
+prediction service. Due to H2O prerequisites, this must be on Mac or Linux and using python 3.6.
+```
+conda create -n daimodel python=3.6
+conda activate daimodel
+```
+
+2. Install the common and model SDK Certifai packages, where `toolkit` is the
+path to where you unzipped the Certifai Toolkit. We will use these packages
+to run the model as a prediction service that Certifai will use:
+```
+pip install toolkit/packages/all/*common*.zip
+pip install toolkit/packages/all/*model*.zip
+```
+
+3. Download and install the MOJO Python runtime from H2O, which is needed to run
+the Mojo from python. Follow the [instructions](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/scoring-pipeline-cpp.html#downloading-the-scoring-pipeline-runtimes) to download and install the runtime into your `daimodel` environment.
+
+4. Unzip the MOJO Scoring Pipeline that you downloaded at the end of your
+experiment, and copy `mojo.pipeline` into this folder.
```

Comment:
`mojo.pipeline` should be renamed to `pipeline.mojo`

This should also be changed for `h2o_dai_german_credit` usecase